autorandr.py

```diff
@@ -852,7 +852,11 @@ def apply_configuration(new_configuration, current_configuration, dry_run=False)
     # Enable the remaining outputs in pairs of two operations
     operations = disable_outputs + enable_outputs
     for index in range(0, len(operations), 2):
-        argv = base_argv + list(chain.from_iterable(operations[index:index + 2]))
+        argv = base_argv
```

Comment:
This creates a reference to `base_argv`. The `+=` below changes `base_argv` itself!

Reply:
Oops!  Yes.  I'm out of practice in mutable collections.